internal/wasm/store.go

```diff
@@ -126,6 +126,9 @@ type (
 		aliases []string
 		// Definitions is derived from *Module, and is constructed during compilation phrase.
 		Definitions []FunctionDefinition
+
+		// globalsView holds a read-only view of all globals for experimental.InteralModule.
+		globalsView []api.Global
```

Comment:
I would like to avoid allocating this slice regardless of the usage of the experimental feature